js/components/controlBar.js

```diff
@@ -727,14 +696,44 @@ var ControlBar = createReactClass({
               null
           }
           height={this.props.skinConfig.controlBar.logo.height} />
+      ),
+
+      skipControls: (
+        <SkipControls
+          key={CONSTANTS.CONTROL_BAR_KEYS.SKIP_CONTROLS}
+          buttonConfig={Utils.getPropertyValue(
+            this.props.skinConfig,
+            'skipControls.buttonsWithPlayPause',
```

Comment:
Is this the same for both audio only and the regular player?

Reply:
Sorry, nevermind. Didn't realize that this is the control bar

Reply:
Yep, this is the control bar version of the skip controls, with the play/pause icon. This will not affect the regular skip controls shown on the playing/pause screens. Currently the control bar skip controls is only used for the audio only player.